.github/workflows/deploy.yml

```diff
@@ -0,0 +1,93 @@
+on:
+  workflow_call:
+    inputs:
+      image_version:
+        required: false
+        type: string
+        description: "Docker image version to pull for deployment"
+        default: "latest"
+      branch:
+        required: false
+        description: "Branch to pull in instance deployment"
+        type: string
+        default: "main"
+      volume_size:
+        required: false
+        description: "Size in GB of instance"
+        type: number
+        default: 10
+      flavor:
+        required: false
+        type: string
+        description: "Flavor of instance"
+        default: "s1-2"
```

Comment:
I see this is a sandbox flavor, I have no knowledge on this, why chose this flavor in particular ?

Reply:
It's only for test, we can change this flavor